### PR TITLE
Show snowfall accumulation

### PR DESCRIPTION
**Pull request by rebfrank**
I think it would be nicer to show this with the "daily" data; right now it's with current data about
precip probability, wind, and humidity. I basically just swapped it with visibility.

### CODE REVIEW

File: public/javascripts/components/weather.js [hunk 1]
```diff
@@ -50,12 +50,12 @@ class Weather extends Component {
                 <td>{Math.round(precipProbability * 100)}%</td>
               </tr>
               <tr>
-                <th>wind</th>
-                <td>{windSpeed} mph {degreesToDirection(windBearing + 180)}</td>
+                <th>snowfall</th>
```
Comment (pifantastic) [inline public/javascripts/components/weather.js]: Can we use `precipType` here?

Comment (rebfrank) [reply public/javascripts/components/weather.js]: Pasting screenshot here cuz hangouts isn't uploading...here is an option putting it in the center...though it sounds like maybe you are thinking it's ok to leave it on the right?

image

Comment (rebfrank) [reply public/javascripts/components/weather.js]: Sorry, that was the wrong spot to paste this...wasn't paying attention.

Comment (pifantastic) [reply public/javascripts/components/weather.js]: We could repurpose the allergies section (which no longer works), e.g.:

Screen Shot 2019-03-12 at 10 09 01 PM

**Comment by rebfrank** [comment]
Playing with the summary:

image

**Pull request merged**